python/src/scipp/neutron/instrument_view.py

```diff
@@ -395,8 +397,13 @@ def change_projection(self, owner):
                     self.det_pos[:, permutations[axis][0]] /
                     np.sqrt(self.det_pos[:, 0]**2 + self.det_pos[:, 1]**2 +
                             self.det_pos[:, 2]**2))
-        self.axes_helper.visible = projection == "3D"
-        self.camera.position = [0, 0, 5]
+
+        if projection == "3D":
+            self.axes_helper.visible = True
+            self.camera.position = [self.camera_pos] * 3
+        else:
+            self.axes_helper.visible = False
+            self.camera.position = [0, 0, self.camera_pos]
```

Comment:
Are the changes to the instrument view here intentional?

Reply:
Yes, it’s a better guess for initial camera position